### ISSUE TITLE
allennlp install error

### ISSUE THREAD

**Issue by alikabeer32** [allenai/allennlp#4049]
while intalling using
`pip install allennlp`

I suddenly get this:

```
Collecting torch>=1.2.0 (from allennlp)
  ERROR: Could not find a version that satisfies the requirement torch>=1.2.0 (from allennlp) (from versions: 0.1.2, 0.1.2.post1, 0.1.2.post2)
ERROR: No matching distribution found for torch>=1.2.0 (from allennlp)
```

**Comment by schmmd** [comment]
@alikabeer32 are you using Python 3.6 or 3.7?

**Comment by alikabeer32** [comment]
> @alikabeer32 are you using Python 3.6 or 3.7?

3.7

But I fixed it by installing torch 1.2.0 using :
`pip install torch==1.2.0 -f https://download.pytorch.org/whl/cu100/torch_stable.html`
